addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -314,7 +310,34 @@ odoo.define('website.tour.form_editor', function (require) {
         {
             content:  "Wait reloading...",
             trigger:  "html:not(:has(#completlyloaded)) div",
-        }
+        },
+        {
+            content: 'Enter in edit mode again',
+            trigger: 'a[data-action="edit"]',
+            run: 'click',
+        },
+        {
+            content: 'Click on the submit button',
+            trigger: '.s_website_form_send',
+            extra_trigger: 'button[data-action="save"]',
+            run: 'click',
+        },
+        {
+            content: 'Change the Recipient Email',
+            trigger: '[data-field-name="email_to"] input',
+            run: 'text test@test.test',
+        },
+        {
+            content: 'Save the page',
+            trigger: 'button[data-action=save]',
+            run: 'click',
+        },
+        {
+            content: 'Verify that the recipient email has been saved',
+            trigger: 'a[data-action="edit"]',
+            // We have to this that way because the input type = hidden.
+            extra_trigger: 'form:has(input[name="email_to"][value="test@test.test"])',
```

Comment:
```suggestion
            trigger: 'body:not(.editor_enable)',
            // We have to this that way because the input type = hidden.
            extra_trigger: 'form:has(input[name="email_to"][value="test@test.test"])',
```
Nitpicking but that's the correct way to wait for the page to be reloaded, I prefer we stick to the same way everywhere, there is already a few different occurences, like the one a few lines above :grimacing: (but it is a 6y old commit so it's normal)

```js
// Save the page
        {
            trigger: 'body',
            run: function () {
                $('body').append('<div id="completlyloaded"></div>');
            },
        },
        {
            content:  "Save the page",
            trigger:  "button[data-action=save]",
        },
        {
            content:  "Wait reloading...",
            trigger:  "html:not(:has(#completlyloaded)) div",
        },
 ```